### PR TITLE
Move askama_mendes integration into Askama repo

### PR DESCRIPTION
**Pull request by djc**
This previously lived in the mendes repo, which was weird since there's also a part that lives in the code generator.

### CODE REVIEW

File: askama_mendes/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[package]
+name = "askama_mendes"
+version = "0.1.0"
+authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
```
Comment (Kijewski) [inline askama_mendes/Cargo.toml:4]: In #562 you removed the author fields.

Comment (djc) [reply askama_mendes/Cargo.toml:4]: Yeah, but this PR is independent of that one. Depending on which one lands first, I'll fix up the other one.

Comment (Kijewski) [reply askama_mendes/Cargo.toml:4]: Hehe, now this is the second PR to land. :) Approved after you cut this one line.

**Review by Kijewski — CHANGES_REQUESTED** [review]

**Comment by Kijewski** [comment]
Hmm, the debug message for errors was changed. It now includes leading spaces for no obvious reason.

**Review by Kijewski — APPROVED** [review]

**Pull request merged**